code/game/objects/items/weapons/twohanded.dm

```diff
@@ -624,4 +624,57 @@
 	sharpness = IS_SHARP
 
 /obj/item/weapon/twohanded/bonespear/update_icon()
-		icon_state = "bone_spear[wielded]"
+	icon_state = "bone_spear[wielded]"
+
+/*
+ * Sky Bulge (Gae Bolg, tradtional dragoon lance from many FF games.)
+ */
+/obj/item/weapon/twohanded/skybulge	//Copy+paste job from bonespear because no explosions.
+	icon_state = "sky_bulge0"
+	name = "Sky Bulge"
+	desc = "A legendary stick with a very pointy tip. Looks to be used for throwing. And jumping. Can be stubborn if you throw too much." //TODO: Be funnier.
+	force = 10 //This thing aint for robusting.
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = SLOT_BACK
+	force_unwielded = 10
+	force_wielded = 18					//Same as regular spear. This is a utility weapon.
+	throwforce = 24						//And that utility is throwing. 24 so it takes 5 hits instead of 4.
+	throw_speed = 4
+	embedded_impact_pain_multiplier = 0	//If you somehow embed this, it's not going to hurt.
+	armour_penetration = 15				//Same as Bone Spear
+	embed_chance = 0					//Would ruin the whole theme of the thing.
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored", "lanced") //Added lanced for flavour.
+	sharpness = IS_SHARP
+	var/maxdist = 16
+	var/throw_cooldown = 0				//Should equate to half a second. Not supposed to be varedited.
+
+/obj/item/weapon/twohanded/skybulge/update_icon()
+	icon_state = "sky_bulge[wielded]"
+
+/obj/item/weapon/twohanded/skybulge/throw_at()  //Throw cooldown and offhand-proofing.
+	if(throw_cooldown > world.time)
+		var/mob/user = thrownby
+		user.put_in_hands(src)
+		user.visible_message("<span class='warning'>But it doesn't leave [user]'s hand!")	//Always let the user know input went through and all that.
```

Comment:
Nothing else I can think of has a failure message when thrown, it's kinda weird; also it means someone can just spam this message with hotkeys. My suggestion would be to axe it.